pkg/replication/replication_mode.go

```diff
@@ -251,10 +251,10 @@ func (m *ModeManager) drSwitchToAsyncWait(availableStores []uint64) error {
 	return nil
 }
 
-func (m *ModeManager) drSwitchToAsync(availableStores []uint64) error {
+func (m *ModeManager) drSwitchToAsync(availableStores []uint64) {
 	m.Lock()
 	defer m.Unlock()
-	return m.drSwitchToAsyncWithLock(availableStores)
+	_ = m.drSwitchToAsyncWithLock(availableStores)
```

Comment:
Why change it?

Reply:
The upper-level call did not use its error, it logs all errors in its implementation, so I ignored the error here.

Reply:
Since this file was also ignored in the golangci.yml, I will revert this change and discuss how to fix it here in the subsequent PRs.